<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyCanvas" name="myCanvas" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>29</y>
      <width>650</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pencilButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ołówek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sprayButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Spray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lineButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Linia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rectangleButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Prostokąt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rValueSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="gValueSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="bValueSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="brushColorLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>543</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Zmień kolor pędzla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circleButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Okrąg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filledCircleButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Koło</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ellipseButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Elipsa</string>
    </property>
   </widget>
   <widget class="QLabel" name="ellipseSlopeLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Zmień nachylenie elipsy</string>
    </property>
   </widget>
   <widget class="QSlider" name="ellipseSlopeSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bezierCurveButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Krzywe Beziera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSplineCurveButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Krzywe B-sklejane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floodFillButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypełnij</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaneLineButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>620</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ScanLine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morphologyOperationsButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Operacje morfologiczne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorModelsButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>620</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modele barw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blendModesButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>620</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blend modes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transformations2DButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>620</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformacje 2D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="texturingButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Teksturowanie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="curveMenu">
    <property name="title">
     <string>Krzywe</string>
    </property>
    <addaction name="bezierCurveAction"/>
    <addaction name="bSplineCurveAction"/>
   </widget>
   <widget class="QMenu" name="menuScaneLine">
    <property name="title">
     <string>ScanLine</string>
    </property>
    <addaction name="scaneLineAction"/>
   </widget>
   <widget class="QMenu" name="menuOperacje_morfologiczne">
    <property name="title">
     <string>Operacje morfologiczne</string>
    </property>
    <addaction name="morphologyOperationsAction"/>
   </widget>
   <widget class="QMenu" name="menuModele_barw">
    <property name="title">
     <string>Modele barw</string>
    </property>
    <addaction name="colorModelsAction"/>
    <addaction name="imageColorModelAction"/>
   </widget>
   <widget class="QMenu" name="menuBlend_modes">
    <property name="title">
     <string>Blend modes</string>
    </property>
    <addaction name="blendModesAction"/>
   </widget>
   <widget class="QMenu" name="menuTransformacje">
    <property name="title">
     <string>Transformacje</string>
    </property>
    <addaction name="transformations2DAction"/>
   </widget>
   <widget class="QMenu" name="menuTexturing">
    <property name="title">
     <string>Texturing</string>
    </property>
    <addaction name="texturingAction"/>
   </widget>
   <addaction name="curveMenu"/>
   <addaction name="menuScaneLine"/>
   <addaction name="menuOperacje_morfologiczne"/>
   <addaction name="menuModele_barw"/>
   <addaction name="menuBlend_modes"/>
   <addaction name="menuTransformacje"/>
   <addaction name="menuTexturing"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="bezierCurveAction">
   <property name="text">
    <string>Beziera</string>
   </property>
  </action>
  <action name="bSplineCurveAction">
   <property name="text">
    <string>B-sklejone</string>
   </property>
  </action>
  <action name="scaneLineAction">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
  <action name="morphologyOperationsAction">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
  <action name="colorModelsAction">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
  <action name="imageColorModelAction">
   <property name="text">
    <string>Na obrazku</string>
   </property>
  </action>
  <action name="blendModesAction">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
  <action name="transformations2DAction">
   <property name="text">
    <string>2D</string>
   </property>
  </action>
  <action name="texturingAction">
   <property name="text">
    <string>Uruchom</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyCanvas</class>
   <extends>QWidget</extends>
   <header>mycanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
